src/main/webapp/dev/DatepickerJsTest.es6

```diff
@@ -0,0 +1,42 @@
+/* global
+triggerDatepickerOnClick:false, getMaxDateForVisibleDate:false
+getMinDateForPublishDate:false
+*/
+
+QUnit.module('datepicker.js');
+
+QUnit.test('triggerDatepickerOnClick(datepickerDivs)', (assert) => {
+    assert.expect(1);
+
+    $('#date-picker-div').datepicker();
+    triggerDatepickerOnClick([$('#date-picker-div')]);
+    $('#date-picker-div').click();
+
+    assert.equal($('.ui-datepicker-calendar:visible').length, 1, 'Displays datepickers after trigger');
+});
+
+QUnit.test('getMaxDateForVisibleDate(startDate, publishDate)', (assert) => {
+    assert.expect(5);
+
+    const startDate = new Date(2017, 3, 19, 2, 31, 0, 0);
+    const publishDate = new Date(2017, 3, 19, 2, 30, 0, 0);
```

Comment:
The terms `startDate` and `publishDate` seem a bit strange here, since the function definition is `function getMaxDateForVisibleDate(startDate, publishDate) { ... }`, but you're calling them in reverse order in one of the tests.

Perhaps name them `laterDate` and `earlierDate` instead?